packages/client/hmi-client/src/components/workflow/ops/interventions/tera-interventions-drilldown.vue

```diff
@@ -0,0 +1,395 @@
+	getInterventionPolicyById,
+	updateInterventionPolicy
+} from '@/services/intervention-policy';
+import Accordion from 'primevue/accordion';
+import AccordionTab from 'primevue/accordiontab';
+import Textarea from 'primevue/textarea';
+import EmptySeed from '@/assets/images/lottie-empty-seed.json';
+import { Vue3Lottie } from 'vue3-lottie';
+import TeraInterventionCard from './tera-intervention-card.vue';
+import { InterventionsOperation, InterventionsState } from './tera-interventions-operation';
+import TeraInterventionsPolicyCard from './tera-interventions-policy-card.vue';
+
+const props = defineProps<{
+	node: WorkflowNode<InterventionsState>;
+}>();
+const emit = defineEmits([
+	'close',
+	'update-state',
+	'select-output',
+	'append-output',
+	'update-output-port'
+]);
+
+const confirm = useConfirm();
+interface BasicKnobs {
+	transientInterventionPolicy: InterventionPolicy;
+}
+
+const knobs = ref<BasicKnobs>({
+	transientInterventionPolicy: {
+		name: '',
+		description: '',
+		modelId: '',
+		interventions: []
+	}
+});
+
+const isSidebarOpen = ref(true);
+const filterInterventionsText = ref('');
+const model = ref<Model | null>(null);
+const isFetchingPolicies = ref(false);
+const interventionsPolicyList = ref<InterventionPolicy[]>([]);
+const interventionPoliciesFiltered = computed(() =>
+	interventionsPolicyList.value.filter((policy) =>
+		policy.name?.toLowerCase().includes(filterInterventionsText.value.toLowerCase())
+	)
+);
+const selectedOutputId = ref<string>('');
+const selectedPolicy = ref<InterventionPolicy | null>(null);
+
+const newDescription = ref('');
+const isEditingDescription = ref(false);
+
+const parameterOptions = computed(() => {
+	if (!model.value) return [];
+	return getParameters(model.value).map((parameter) => ({
+		label: parameter.name ?? parameter.id,
+		value: parameter.id
+	}));
+});
+
+const groupedOutputParameters = computed(() =>
+	groupBy(selectedPolicy.value?.interventions, 'appliedTo')
+);
+
+const initialize = async () => {
+	const state = props.node.state;
+	const modelId = props.node.inputs[0].value?.[0];
+	if (!modelId) return;
+
+	// fetch intervention policies
+	await fetchInterventionPolicies(modelId);
+
+	model.value = await getModel(modelId);
+	if (state.transientInterventionPolicy?.id) {
+		// copy the state into the knobs if it exists
+		selectedPolicy.value = await getInterventionPolicyById(state.transientInterventionPolicy.id);
+		knobs.value.transientInterventionPolicy = cloneDeep(state.transientInterventionPolicy);
+	} else {
+		knobs.value.transientInterventionPolicy = cloneDeep(state.transientInterventionPolicy);
+	}
+};
+
+const applyInterventionPolicy = (interventionPolicy: InterventionPolicy) => {
+	const state = cloneDeep(props.node.state);
+	knobs.value.transientInterventionPolicy = cloneDeep(interventionPolicy);
+
+	const listOfPolicyIds: string[] = props.node.outputs.map((output) => output.value?.[0]);
+	// Update output port:
+	if (!interventionPolicy?.id) {
```

Comment:
Should this check not be before 248's clone?
This comment update output port does not apply to this specific if